DependencyInjection/Compiler/ThemeCompilerPass.php

```diff
@@ -30,10 +30,14 @@ public function process(ContainerBuilder $container)
 
         $twigFilesystemLoaderDefinition = $container->findDefinition('twig.loader.filesystem');
         $twigFilesystemLoaderDefinition->setClass($container->getParameter('liip_theme.filesystem_loader.class'));
-        $twigFilesystemLoaderDefinition->setArguments(array(
-            $container->getDefinition('liip_theme.templating_locator'),
-            $container->getDefinition('templating.filename_parser')
-        ));
+
+        if (false === $container->has('templating')) {
+            $twigFilesystemLoaderDefinition->setArguments(array(
```

Comment:
What about using `replaceArgument()` instead to not break in case the second argument was changed somewhere else?

Reply:
Great point @xabbuh, I will use the method `replaceArgument()`.
However if the argument number for `twig.loader.filesystem` service change, the `replaceArgument()` method will throw an `OutOfBoundsException`.